baseapp-auth/setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = baseapp_auth
-version = 0.3.1
+version = 0.3.2
```

Comment:
shouldn't we bump the baseapp-core instead?

Reply:
Yes, you're right. I'll bump baseapp-core instead